pylint_odoo/checkers/no_modules.py

```diff
@@ -390,6 +390,12 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def _convert_value_to_name(self, attribute):
+        """Convert 'Field Case SENSITIVE' to 'field_case_sENSITIVE' """
+        parts = [(part[0].lower() + part[1:]) for part in attribute.split(' ')]
```

Comment:
What about use `title` and `capitalize` builtin functions?
<img width="525" alt="screen shot 2017-01-24 at 3 20 42 pm" src="https://cloud.githubusercontent.com/assets/6644187/22267061/ee821990-e247-11e6-927f-e7cefa676f11.png">